packages/coin-support-tron/src/services/validators/index.ts

```diff
@@ -0,0 +1,107 @@
+import { z } from 'zod';
```

Comment:
Don't keep the API schema strict, it can affect the backward compatibility of the API.

- It should allow for unidentified values
- It should work even if some of the values are `undefined` or `null`

Reply:
converted type of unused properties from response object to nullish.